ios/UARCTModule/UrbanAirshipReactModule.m

```diff
@@ -52,6 +52,11 @@ - (RCTBridge *)bridge {
     [UAirship push].userPushNotificationsEnabled = enabled;
 }
 
+
```

Comment:
Is there a reason why these methods are missing comment headers?

Reply:
None of them are defined

Reply:
I saw that. It's weird that they are defined in the Android code, but not the iOS code.